<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCMainWidget</class>
 <widget class="QWidget" name="FCMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCMainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="layout" stretch="0,0">
   <item>
    <widget class="FCFileInfoWidget" name="fiWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="opWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="seekLayout">
          <item>
           <widget class="QLineEdit" name="seekEdit">
            <property name="placeholderText">
             <string>跳转到...(秒)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fastSeekBtn">
            <property name="text">
             <string>快速跳转</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exactSeekBtn">
            <property name="text">
             <string>精确跳转</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="setStartBtn">
          <property name="text">
           <string>设为起始帧</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="durationEdit">
            <property name="placeholderText">
             <string>时长</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="durationUnitComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>文本：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>字号：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fontSizeEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addTextBtn">
          <property name="text">
           <string>添加文字</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="gifBtn">
        <property name="text">
         <string>保存GIF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FCFileInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">fcfileinfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
